purchase_mass_mail/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,3 @@
+This module add a server action in the menú action in the purchase order tree and form views,
+in the tree view of the purchase order you can select some purchase orders and click in
+the action named "Send email to selected purchases" , will open a wizard with the default template and you can send this email to all purchase orders selected
```

Comment:
I'm rewording the same sentence to make it easier to read:

```suggestion
This module adds a contextual action in the purchase order views, named "Send email to selected purchases" .
It will open a wizard with the default template and you can send this email to all purchase orders selected.
```